### DISCUSSION TITLE
How to affect a simple velocity to an object ?

### DISCUSSION THREAD

**Discussion by Ayfri** [dyn4j/dyn4j#263] [Q&A]
Hi, I'm struggling a lot on how to simply affect a constant speed to an object with also a "jump" feature. I don't understand at all what I should do, when I apply a force, first I have to apply a force of `new Force(0f, -1000000000f)` to have my object to jump a few meters, and also it stops any horizontal force. If I apply `linearVelocity.setX(200.0)` every frame, the gravity stops working. If I apply it only in the constructor of my Body, my object slows done continuously.

I'm using the graphical library Processing 4 with Kotlin.

**Comment by wnbittle** [comment]
There's a sample app called `Platformer` in the samples repo that tries to give some ideas on jumping.

Another thought I had was to look at the mass of the body you are trying to effect.  If it's high mass, you'll need a lot of force.

Applying a force should not stop horizontal motion. Maybe you are hitting one of default limits in `Settings`.  I think there's a maximum linear velocity setting in there.
